app/code/Magento/InventoryAdminUi/Test/Mftf/Test/VerifyThatMessageOnly0LeftNotAppearedOnSimpleProductPageOnFrontendTest.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="VerifyThatMessageOnly0LeftNotAppearedOnSimpleProductPageOnFrontendTest">
+        <annotations>
+            <stories value="Simple Product created on Test source with Qty = 0 and status 'Out of stock"/>
+            <title value="Sets Only X left Threshold = 0 in Configuration, not provide message 'Only 0 left"/>
+            <description value="Verify that message 'Only 0 left' not appeared on Simple product page on frontend"/>
+            <severity value="CRITICAL"/>
+            <group value="msi"/>
+            <group value="multi_mode"/>
+        </annotations>
+
+        <before>
+            <comment userInput="Turn on 'out of stock' config" stepKey="onOutOfStockConfig"/>
+            <magentoCLI command="config:set cataloginventory/options/show_out_of_stock 1"
```

Comment:
Please avoid hardcoding commands. It would be best to introduce a Data file (you may use `app/code/Magento/Downloadable/Test/Mftf/Data/CatalogConfigData.xml` as a reference) and use templates in `magentoCLI` node.

Reply:
ok